tests/lib/rules/indent.js

```diff
@@ -2971,6 +2971,49 @@ ruleTester.run("indent", rule, {
             "}",
             options: [2, {FunctionExpression: {parameters: 3}}],
             errors: expectedErrors([3, 8, 10, "Identifier"])
-        }
+        },
+        {
+            code: `
+{
+    try {
+    }
+catch (err) {
+    }
+finally {
+    }
+}
```

Comment:
nit: I think we should stick to string concat here because this makes it heard to read in this particular scenario. (or maybe its just me)